src/android/facebookconnect.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
 <resources>
+    <string name="fb_app_id">1614119758827093</string>
+    <string name="fb_app_name">app.hurdlr.com</string>
```

Comment:
Oh yea, I was going to take it out but completely forgot about it. Good find. I will remove it and commit